etc/alignak.cfg

```diff
@@ -124,14 +124,6 @@ use_ssl=0
 #hard_ssl_name_check=0
 
 
-# kernel.alignak.io communication channel. Create an account to http://shinken.io
-# and look at your profile to fill this.
-#api_key=
-#secret=
-# if you need an http proxy to exchange with kernel.alignak.io
-#http_proxy=
```

Comment:
Perhaps we can keep _api_key_, _secret_ and _http_proxy_ because we use it for statsd